src/items/itemsLandingPage.component.tsx

```diff
@@ -24,10 +29,32 @@ import { useGetItem } from '../api/items';
 import { useGetManufacturer } from '../api/manufacturers';
 import { useGetSystem } from '../api/systems';
 import { useNavigateToCatalogue } from '../catalogue/catalogue.component';
-import { formatDateTimeStrings } from '../utils';
+import PlaceholderImage from '../common/placeholderImage.component';
+import {
+  a11yProps,
+  formatDateTimeStrings,
+  StyledTab,
+  TabPanel,
+} from '../utils';
 import Breadcrumbs from '../view/breadcrumbs.component';
 import ItemDialog from './itemDialog.component';
 
+const ITEMS_TAB_VALUES = [
+  'Information',
+  'Gallery',
+  'Attachments',
+  'Notes',
+] as const;
+
+const iconMapping: Record<ItemsTabValue, React.ReactElement> = {
+  Information: <InfoOutlinedIcon />,
+  Gallery: <CollectionsOutlinedIcon />,
+  Attachments: <AttachmentOutlinedIcon />,
+  Notes: <NotesIcon />,
+};
+
+type ItemsTabValue = (typeof ITEMS_TAB_VALUES)[number];
```

Comment:
```suggestion
const ITEMS_TAB_VALUES = [
  'Information',
  'Gallery',
  'Attachments',
  'Notes',
] as const;

type ItemsTabValue = (typeof ITEMS_TAB_VALUES)[number];

const iconMapping: Record<ItemsTabValue, React.ReactElement> = {
  Information: <InfoOutlinedIcon />,
  Gallery: <CollectionsOutlinedIcon />,
  Attachments: <AttachmentOutlinedIcon />,
  Notes: <NotesIcon />,
};
```
So that its in the order its used.